site/_data/master-toc.yml

```diff
@@ -45,18 +53,14 @@ toc:
         url: /debugging-restores
       - page: Troubleshoot Restic
         url: /restic#troubleshooting
-  - title: Customize
+  - title: Contribute
     subfolderitems:
       - page: Build from source
         url: /build-from-source
-      - page: Run in any namespace
-        url: /namespace
-      - page: Extend
-        url: /extend
-      - page: Extend with plugins
-        url: /plugins
-      - page: Extend with hooks
-        url: /hooks
+      - page: Run locally
+        url: /run-locally
+      - page: Add code
```

Comment:
The title of this page is a little confusing to me. I wonder if this should be called "Developer guide" or something else, as the content seems to be general development information.